### PR TITLE
feat(registration-api-integration ): add user service

### PR DESCRIPTION
**Pull request by YgRamos9**
## Summary
- implement registration API

## Screenshot
Captura de Pantalla 2022-08-01 a la(s) 6 29 25 p m

## Trello Card
https://trello.com/c/SseFp8Xv/3-registration-api-integration

### CODE REVIEW

File: angular/src/app/components/form-container/form-container.component.ts [hunk 1]
```diff
@@ -11,17 +11,17 @@ import { IFields } from 'src/app/interfaces/global.interface';
 export class FormContainerComponent {
   @Input() formFields!: IFields[];
   @Input() buttonSend!: string;
-  @Output() handleOnSubmit = new EventEmitter<object>();
+  @Output() handleOnSubmit = new EventEmitter<IUser>();
```
Comment (miguelhenaoa) [inline angular/src/app/components/form-container/form-container.component.ts:14]: 💯

File: angular/src/app/screens/sign-up/sign-up.component.ts [hunk 2]
```diff
@@ -10,7 +13,15 @@ export class SignUpComponent {
...
 
-  handleOnSubmit(value: object) {
-    console.log(value);
+  constructor(private userService: UserService) {}
+
+  handleOnSubmit(value: IUser): void {
+    this.userService
+      .createUser(value)
+      .subscribe((response: HttpResponse<Object>) => {
+        if (response.status === 201) {
```
Comment (joacomiguens) [inline angular/src/app/screens/sign-up/sign-up.component.ts]: use a global constant for 201 status code

**Review by joacomiguens — CHANGES_REQUESTED** [review]
Great work! Just a minor comment 😄

File: angular/src/app/constants/codeRequest.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export enum Status {
+  success = 1,
```
Comment (joacomiguens) [inline angular/src/app/constants/codeRequest.ts]: I think it should be 201 🤔

**Review by joacomiguens — APPROVED** [review]

**Review by miguelhenaoa — APPROVED** [review]

**Pull request merged**